src/tests/JIT/HardwareIntrinsics/X86/Avx512F/Program.Avx512F.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+
+namespace JIT.HardwareIntrinsics.X86._Avx512F
+{
+    public static partial class Program
```

Comment:
Is this how these tests are organized or it's a dead code?

Reply:
Its a stub file that all the generated test projects have at the moment. We could probably remove it, but it's part of the existing "template" so I copied it here as well.